.tugboat/config.yml

```diff
@@ -43,6 +43,7 @@ services:
         - grep -RiIl "accessibility\.civicactions\.com" _site/. | xargs sed -i "s/accessibility\.civicactions\.com/${TUGBOAT_PREVIEW_REF}-${TUGBOAT_SERVICE_TOKEN}\.tugboatqa\.com/g"
 
         # Inject .htaccess file to mimic GitHub pages behavior with URLs.
+        # Solution from https://serverfault.com/questions/989333/using-apache-rewrite-rules-in-htaccess-to-remove-html-causing-a-500-error.
```

Comment:
Can you add more context here about why this comment is added?

Reply:
Attributing where I got the solution for the rewrite rules in that config.